src/dev/ci_setup/get_percy_env.js

```diff
@@ -25,5 +25,8 @@ const shortCommit = commit.slice(0, 8);
 
 const isPr = process.env.JOB_NAME.includes('elastic+kibana+pull-request');
 
+const { stdout: branch } = execa.sync('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
```

Comment:
Any reason for this not to be what is tracked in the package.json? https://github.com/elastic/kibana/blob/master/package.json#L15

Reply:
I want to keep it separate so that feature branches can deviate from the base snapshots.